ui/src/app/cohort-search/utils.ts

```diff
@@ -25,12 +26,21 @@ export function nameDisplay(parameter): string {
   } else {
     let name = stripHtml(parameter.name);
     if (parameter.type === CriteriaType.ETHNICITY || parameter.type === CriteriaType.RACE) {
-      name = new TitleCasePipe().transform(name);
+      name = toTitleCase(name);
     }
     return name;
   }
 }
 
+export function toTitleCase(str: string): string {
+  const tokens = str.toLowerCase().split(' ');
+  const titleCasedTokens = fp.flow(
+    fp.filter((token: string) => token.length > 0),
+    fp.map((token: string) => token.charAt[0].toUpperCase() + token.slice(1))
+  )(tokens);
+  return titleCasedTokens.join(' ');
```

Comment:
i think you should be able to do `_.startCase(_.toLower(str));`

Reply:
ooh, neat. Thanks!